<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>578</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>578</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>578</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(119, 161, 161);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/i.webp);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="text_1">
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>определиться с профессией бывает очень сложно</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_2">
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>поэтому вам предложено пройти тест на профориентацию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_3">
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>либо просмотреть варианты профессий и специальностей к ним, </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_4">
       <property name="font">
        <font>
         <family>Nirmala UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>а также статьи по каждой из них</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_test">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(145, 182, 182);
	color: white;
	border-width: 2px;
	border-radius: 15px;
}
</string>
    </property>
    <property name="text">
     <string>ПРОЙТИ ТЕСТИРОВАНИЕ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_prof">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(145, 182, 182);
	color: white;
	border-width: 2px;
	border-radius: 15px;
}

QPushbutton:pressed {
	background-color: white

}</string>
    </property>
    <property name="text">
     <string>ПРОСМОТРЕТЬ СПИСОК ПРОФЕССИЙ</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="D:/prof.qrc"/>
 </resources>
 <connections/>
</ui>
